packages/ember-views/lib/compat/metamorph_view.js

```diff
@@ -26,9 +26,12 @@ export var _Metamorph = Mixin.create({
   init() {
     this._super.apply(this, arguments);
     Ember.deprecate('Supplying a tagName to Metamorph views is unreliable and is deprecated.' +
-                    ' You may be setting the tagName on a Handlebars helper that creates a Metamorph.', !this.tagName);
+                    ' You may be setting the tagName on a Handlebars helper that creates a Metamorph.',
+                    !this.tagName,
+                    { id: 'ember-views.metamorph-tag-name', until: '3.0.0' });
```

Comment:
Lets tweak this to 2.4.0, because it will be unsupported after ember-legacy-views drops support (which is 2.4).